src/pages/token/LaunchIncentivesPanel.tsx

```diff
@@ -50,7 +44,7 @@ export const LaunchIncentivesPanel = ({ className }: { className?: string }) =>
   }, [dispatch]);
 
   const isSept2025Rewards = useStatsigGateValue(StatsigFlags.ffSeptember2025Rewards);
-  if (isSept2025Rewards) {
+  if (!isSept2025Rewards) {
     return <September2025RewardsPanel />;
   }
```

Comment:
hmmm should we just remove this? Why show the rewards panel when the flag goes off?